### PR TITLE
Add meeting module

### PR DESCRIPTION
**Pull request by ele828**

### CODE REVIEW

File: src/modules/Meeting/index.js [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+import getMeetingReducer, { getMeetingStorageReducer } from './getMeetingReducer';
+
+export const UTC_TIMEZONE_ID = '1';
+export const MeetingType = {
+  SCHEDULED: 'Scheduled',
+  RECURRING: 'Recurring',
+  INSTANT: 'Instant',
+};
+
+// Basic default meeting type infomation
```
Comment (embbnux) [inline src/modules/Meeting/index.js]: typo information?

File: src/modules/Meeting/index.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+      topic,
+      password,
+      schedule,
+      _requireMeetingPassword,
+    } = meeting;
+    const errors = new MeetingErrors();
+    if (topic.length <= 0) {
+      errors.push(meetingStatus.emptyTopic);
+    }
+    if (meeting._requireMeetingPassword && (!password || password.length <= 0)) {
```
Comment (embbnux) [inline src/modules/Meeting/index.js]: Please use _requireMeetingPassword

**Comment by embbnux** [comment]
Please remove empty file from your commit.

File: src/modules/Meeting/index.js (hunk 1 above)
Comment (ele828) [reply src/modules/Meeting/index.js]: fixed

File: src/modules/Meeting/index.js (hunk 2 above)
Comment (ele828) [reply src/modules/Meeting/index.js]: fixed

**Pull request merged**